QueryDemo.ipynb

```diff
@@ -0,0 +1,3670 @@
+      "        }\n",
+      "    ],\n",
+      "    \"SpectralTypes\": [\n",
+      "        {\n",
+      "            \"spectral_type_string\": \"T3\",\n",
+      "            \"spectral_type_code\": 83.0,\n",
+      "            \"spectral_type_error\": null,\n",
+      "            \"regime\": \"nir\",\n",
+      "            \"adopted\": null,\n",
+      "            \"photometric\": null,\n",
+      "            \"comments\": null,\n",
+      "            \"reference\": \"Kirk11\"\n",
+      "        }\n",
+      "    ]\n",
+      "}\n",
+      "{'Sources': [{'source': 'WISEPC J033349.34-585618.7', 'ra': 53.4557, 'dec': -58.9384, 'epoch': None, 'equinox': '2000', 'shortname': None, 'reference': 'Kirk11', 'other_references': None, 'comments': None}], 'Names': [{'other_name': 'CWISE J033349.31-585619.8'}, {'other_name': 'VHS J033349.30-585620.2'}, {'other_name': 'WISEPC J033349.34-585618.7'}], 'Parallaxes': [{'parallax': 46.2, 'parallax_error': 3.7, 'adopted': True, 'comments': None, 'reference': 'Kirk21'}], 'Photometry': [{'band': 'IRAC.I1', 'magnitude': 13.59, 'magnitude_error': 0.02, 'telescope': 'Spitzer', 'epoch': None, 'comments': None, 'reference': 'Kirk11'}, {'band': 'IRAC.I2', 'magnitude': 13.3, 'magnitude_error': 0.02, 'telescope': 'Spitzer', 'epoch': None, 'comments': None, 'reference': 'Kirk11'}, {'band': 'VISTA.J', 'magnitude': 15.73, 'magnitude_error': 0.005, 'telescope': 'VISTA', 'epoch': None, 'comments': None, 'reference': 'McMa13'}, {'band': 'WISE.W1', 'magnitude': 14.037, 'magnitude_error': 0.017, 'telescope': 'WISE', 'epoch': None, 'comments': 'Ingested from the Ultracool Sheet. Cross match to CatWISE done by Sanghi et al. 2023 (10.3847/1538-4357/acff66).', 'reference': 'Maro21'}, {'band': 'WISE.W2', 'magnitude': 13.313, 'magnitude_error': 0.013, 'telescope': 'WISE', 'epoch': None, 'comments': 'Ingested from the Ultracool Sheet. Cross match to CatWISE done by Sanghi et al. 2023 (10.3847/1538-4357/acff66).', 'reference': 'Maro21'}, {'band': 'WISE.W3', 'magnitude': 12.22, 'magnitude_error': 0.225, 'telescope': 'WISE', 'epoch': None, 'comments': 'Ingested from the Ultracool Sheet. Cross match to CatWISE done by Sanghi et al. 2023 (10.3847/1538-4357/acff66).', 'reference': 'Cutr13'}], 'ProperMotions': [{'mu_ra': -126.0, 'mu_ra_error': 8.0, 'mu_dec': -607.0, 'mu_dec_error': 7.0, 'adopted': False, 'comments': None, 'reference': 'Kirk11'}, {'mu_ra': -142.7, 'mu_ra_error': 3.8, 'mu_dec': -620.0, 'mu_dec_error': 3.4, 'adopted': True, 'comments': 'Ingested from the Ultracool Sheet. Cross match to CatWISE done by Sanghi et al. 2023 (10.3847/1538-4357/acff66).', 'reference': 'Maro21'}], 'SpectralTypes': [{'spectral_type_string': 'T3', 'spectral_type_code': 83.0, 'spectral_type_error': None, 'regime': 'nir', 'adopted': None, 'photometric': None, 'comments': None, 'reference': 'Kirk11'}]}\n"
+     ]
+    }
+   ],
+   "source": [
+    "data = db.inventory('WISEPC J033349.34-585618.7', pretty_print=True)\n",
+    "print(data) "
```

Comment:
You don't need to add `print(data)` if you already set `pretty_print=True`.    
If you notice, this causes the dictionary to be printed twice, once in a fairly easy to read way and the other as a big blob.